script/hassfest/docker/Dockerfile

```diff
@@ -23,7 +23,7 @@ RUN --mount=from=ghcr.io/astral-sh/uv:0.2.27,source=/uv,target=/bin/uv \
         -c /usr/src/homeassistant/homeassistant/package_constraints.txt \
         -r /usr/src/homeassistant/requirements.txt \
         stdlib-list==0.10.0 pipdeptree==2.23.1 tqdm==4.66.4 ruff==0.6.2 \
-        PyTurboJPEG==1.7.5 ha-ffmpeg==3.2.0 hassil==1.7.4 home-assistant-intents==2024.9.4 mutagen==1.47.0
+        PyTurboJPEG==1.7.5 ha-ffmpeg==3.2.0 hassil==1.7.4 home-assistant-intents==2024.9.4 mutagen==1.47.0 pymicro-vad==1.0.1 pyspeex-noise==1.0.2
```

Comment:
they are dependencies of `assist_pipeline`. The dependencies listed here are automatically gathered by recursively fetching requirements of entity components and its dependencies (in hassfest/docker.py)

Reply:
Is it required for this PR somehow? It sounds like it should just be a separate PR for the Assist pipeline integration.

Reply:
These requirements are not new to Assist Pipeline. Because Assist Satellite is an entity component and it depends on Assist Pipeline, the requirements are pulled into here.

Reply:
Ah, now I understand. Thanks!